_inc/client/at-a-glance/stats.jsx

```diff
@@ -144,32 +153,34 @@ const DashStatsBottom = React.createClass( {
 		<div>
 			<div className="jp-at-a-glance__stats-summary">
 				<div className="jp-at-a-glance__stats-summary-today">
-					<p className="jp-at-a-glance__stat-details">Views today</p>
+					<p className="jp-at-a-glance__stat-details">{ __( 'Views today', { comment: 'Referring to a number of page views' } ) }</p>
 					<h3 className="jp-at-a-glance__stat-number">{ s.viewsToday }</h3>
 				</div>
 				<div className="jp-at-a-glance__stats-summary-bestday">
-					<p className="jp-at-a-glance__stat-details">Best overall day</p>
-					<h3 className="jp-at-a-glance__stat-number">{ s.bestDay.count } Views</h3>
+					<p className="jp-at-a-glance__stat-details">{ __( 'Best overall day', { comment: 'Referring to a number of page views' } ) }</p>
+					<h3 className="jp-at-a-glance__stat-number">{ __( '%(number)s Views', { args: { number: s.bestDay.count } } ) }</h3>
```

Comment:
It's a bit of an edge-case, but this could potentially be `1`, so we should set it up as a proper plural.